src/database.ts

```diff
@@ -1645,7 +1875,7 @@ class Database extends ServiceObject {
    * @example
    * var session = database.session('session-name');
    */
-  session(name?: string) {
+  session(name?: string): Session {
```

Comment:
No need to declare the return signature as it is inferred from the `return` statement here
```suggestion
  session(name?: string) {
```